Kickstarter-iOS/Views/Controllers/ManagePledgeSummaryView.swift

```diff
@@ -0,0 +1,196 @@
+    self.shippingLocationStackView.rac.hidden = self.viewModel.outputs.shippingLocationStackViewIsHidden
+    self.totalAmountLabel.rac.attributedText = self.viewModel.outputs.totalAmountText
+  }
+
+  // MARK: Functions
+
+  private func configureViews() {
+    _ = (self.rootStackView, self)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToMarginsInParent()
+
+    _ = ([self.backerNumberLabel, self.backingDateLabel], self.backerInfoStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.pledgeLabel, self.pledgeAmountLabel], self.pledgeAmountStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.shippingLocationLabel, self.shippingAmountLabel], self.shippingLocationStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([self.totalLabel, self.totalAmountLabel], self.totalAmountStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = ([
+      self.backerInfoStackView,
+      self.pledgeAmountStackView,
+      self.shippingLocationStackView,
+      self.totalAmountStackView
+    ], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+  }
+}
+
+// MARK: Styles
+
+private let amountLabelStyle: LabelStyle = { label in
+  label
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.textAlignment .~ NSTextAlignment.right
+    |> \.isAccessibilityElement .~ true
+    |> \.minimumScaleFactor .~ 0.75
+}
+
+private let backerInfoStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> \.axis .~ .vertical
+}
+
+private let backerNumberLabelStyle: LabelStyle = { label in
+  label
+    |> \.textColor .~ UIColor.ksr_soft_black
+    |> \.font .~ UIFont.ksr_subhead().bolded
+    |> \.adjustsFontForContentSizeCategory .~ true
+}
+
+private let backingDateLabelStyle: LabelStyle = { label in
+  label
+    |> \.font .~ UIFont.ksr_subhead()
+    |> \.textColor .~ UIColor.ksr_dark_grey_500
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.minimumScaleFactor .~ 0.5
+}
+
+private let pledgeLabelStyle: LabelStyle = { label in
+  label
+    |> \.textColor .~ UIColor.ksr_dark_grey_500
+    |> \.font .~ UIFont.ksr_subhead().bolded
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.text %~ { _ in Strings.Pledge() }
+}
+
+private let rootStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> checkoutStackViewStyle
+    |> \.spacing .~ Styles.grid(3)
+}
+
+private let shippingLocationLabelStyle: LabelStyle = { label in
+  label
+    |> \.textColor .~ UIColor.ksr_dark_grey_500
+    |> \.font .~ UIFont.ksr_subhead().bolded
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.minimumScaleFactor .~ 0.5
+}
+
+private let totalLabelStyle: LabelStyle = { label in
+  label
+    |> \.textColor .~ UIColor.black
+    |> \.font .~ UIFont.ksr_subhead().bolded
+    |> \.adjustsFontForContentSizeCategory .~ true
+    |> \.text %~ { _ in "Total amount" }
```

Comment:
Best to use `localizedString` directly until we have the string added.

Reply:
Actually after merging master I noticed that most of the strings were already there!